core/src/main/java/org/pentaho/platform/util/Emailer.java

```diff
@@ -173,7 +178,7 @@ public boolean setup() {
       props.put( "mail.smtp.port", ObjectUtils.toString( service.getEmailConfig().getSmtpPort() ) );
       props.put( "mail.transport.protocol", service.getEmailConfig().getSmtpProtocol() );
       props.put( "mail.smtp.starttls.enable", ObjectUtils.toString( service.getEmailConfig().isUseStartTls() ) );
-      props.put( "mail.smtp.auth", ObjectUtils.toString( service.getEmailConfig().isAuthenticate() ) );
+      //props.put( "mail.smtp.auth", ObjectUtils.toString( service.getEmailConfig().isAuthenticate() ) );
```

Comment:
Consider deleting the line. Otherwise, please leave a comment explaining its relevance.